includes/abstracts/abstract-wc-settings-api.php

```diff
@@ -889,62 +831,49 @@ public function validate_textarea_field( $key ) {
 	 *
 	 * If not set, return "no", otherwise return "yes".
 	 *
-	 * @param  mixed $key
-	 * @since  1.0.0
+	 * @param  string $key
 	 * @return string
 	 */
 	public function validate_checkbox_field( $key ) {
-
-		$status = 'no';
-		$field  = $this->get_field_key( $key );
-
-		if ( isset( $_POST[ $field ] ) && ( 1 == $_POST[ $field ] ) ) {
-			$status = 'yes';
-		}
-
-		return $status;
+		$field_key = $this->get_field_key( $key );
+		return isset( $_POST[ $field_key ] ) && '1' === $_POST[ $field_key ] ? 'yes' : 'no';
 	}
 
 	/**
 	 * Validate Select Field.
 	 *
-	 * Make sure the data is escaped correctly, etc.
-	 *
-	 * @param  mixed $key
-	 * @since  1.0.0
+	 * @param  string $key
 	 * @return string
 	 */
 	public function validate_select_field( $key ) {
-
-		$value = $this->get_option( $key );
-		$field = $this->get_field_key( $key );
-
-		if ( isset( $_POST[ $field ] ) ) {
-			$value = wc_clean( stripslashes( $_POST[ $field ] ) );
-		}
-
-		return $value;
+		$field_key = $this->get_field_key( $key );
+		return isset( $_POST[ $field_key ] ) ? wc_clean( stripslashes( $_POST[ $field_key ] ) ) : $this->get_option( $key );
 	}
 
 	/**
 	 * Validate Multiselect Field.
 	 *
-	 * Make sure the data is escaped correctly, etc.
-	 *
-	 * @param  mixed $key
-	 * @since  1.0.0
+	 * @param  string $key
 	 * @return string
 	 */
 	public function validate_multiselect_field( $key ) {
+		$field_key = $this->get_field_key( $key );
+		return isset( $_POST[ $field_key ] ) ? array_map( 'wc_clean', array_map( 'stripslashes', (array) $_POST[ $field_key ] ) ) : '';
+	}
 
-		$field = $this->get_field_key( $key );
-
-		if ( isset( $_POST[ $field ] ) ) {
-			$value = array_map( 'wc_clean', array_map( 'stripslashes', (array) $_POST[ $field ] ) );
-		} else {
-			$value = '';
-		}
+	/**
+	 * Validate the data on the "Settings" form.
+	 * @deprecated 2.6.0 No longer used
+	 */
+	public function validate_settings_fields( $form_fields = array() ) {}
 
+	/**
+	 * Format settings if needed.
+	 * @deprecated 2.6.0 Unused
+	 * @param  array $value
+	 * @return array
+	 */
+	public function format_settings( $value ) {
 		return $value;
```

Comment:
If this (and the above function) are deprecated, should they call `_deprecated_function`?